cloud/aws/main.tf

```diff
@@ -83,7 +84,7 @@ resource "null_resource" "cockroach-initializer" {
     inline = [
       "bash download_binary.sh cockroach/cockroach ${var.cockroach_sha}",
       "mkdir -p logs",
-      "./cockroach init --logtostderr=true --stores=sdd=data",
+      "./cockroach init --logtostderr=true --stores=${element(split(",", "${var.stores}"), 0)}",
```

Comment:
why `element(..., 0)`? if you provide `--var=stores=ssd=data,hdd=data2`, it will just use `ssd=data`.
You should also make sure that the extra `=` doesn't cause issues (eg: if they just call `strings.Split`, we're hosed)